episodes/filetypes-and-metadata.md

```diff
@@ -0,0 +1,498 @@
+- Inspect and set an image's scale in Napari
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image file formats
+
+Images can be saved in a wide variety of file formats - e.g. you may be familiar 
+with some like .jpg, .png or .tiff. Microscopy images have an especially wide 
+range of options, with hundreds of different formats in use, often specific to 
+particular microscope manufacturers. With this being said, how do we choose 
+which format to use?
+
+## Metadata
+
+First, let's look closer at what gets stored inside an image file.
+
+There are two main things that get stored inside image files: _pixel values_ and 
+_metadata_. We've looked at pixel values in previous episodes -  this is the raw 
+image data as an array of numbers with specific dimensions and data type. The 
+metadata, on the other hand, is a wide variety of additional information about 
+the image and how it was acquired.
+
+For example, let's take a look at the metadata in the 
+'Plate1-Blue-A-12-Scene-3-P3-F2-03.czi' file we downloaded as part of the 
+[setup instructions](../learners/setup.md). To browse the metadata we will use 
+a Napari plugin called [napari-aicsimageio
+](https://www.napari-hub.org/plugins/napari-aicsimageio). This plugin allows a 
+wide variety of file formats to be opened in Napari that aren't supported by 
+default. This plugin was already installed in the [setup instructions
+](../learners/setup.md), so you should be able to start using it straight away.
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+### Napari-aicsimagio installation
+
+You may be wondering why we didn't install this plugin via 
+`Plugins > Install / Uninstall Plugins...` as we showed in the 
+[Image Display episode](./FIXME.md). To support the widest range of file 
+formats, napari-aicsimagio needs integration with Bio-Formats (we'll discuss 
+Bio-Formats later in the episode). Unfortunately, this isn't currently possible 
+via `Plugins > Install / Uninstall Plugins...`, hence why we used 
+[conda to install it
+](https://github.com/AllenCellModeling/napari-aicsimageio#installation).
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+Let's open the 'Plate1-Blue-A-12-Scene-3-P3-F2-03.czi' file by removing any 
+existing image layers, then dragging and dropping it onto the canvas. In the 
+pop-up menu that appears, select 'napari-aicsimageio'. Note this can take a 
+while to open, so give it some time! Alternatively, you can select in Napari's 
+top menu-bar:  
+`File > Open File(s)...`
+
+![](fig/plate1-czi-napari.png){alt="A screenshot of yeast sample data shown in 
+Napari"}
+
+This image is part of a [published dataset on Image Data Resource
+](https://idr.openmicroscopy.org/search/?query=Name:idr0011-ledesmafernandez-dad4/screenD) 
+(accession number idr0011), and comes from the [OME sample data
+](https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/). It is a 3D 
+fluorescence microscopy image of yeast with three channels.
```

Comment:
See my comments on the mutli-dimensional image PR. I think technically this image has 3 layers, each layer being a single channel image. Annoyingly each layer is named as a channel (presumably in the metadata). Once we've decided how we're going to handle naming consistency in the multi-dimension-image episode we need to check here and make sure we are consistent.